src/vector/v.what.rast.label/v.what.rast.label.html

```diff
@@ -0,0 +1,37 @@
+<h2>DESCRIPTION</h2>
+
+<em>v.what.rast.label</em> retrieves raster values and labels from one
+or more given raster maps for each point stored in the input vector map.
+The name of the columns with the raster values and labels are
+respectively<i>ID_name-of-raster-map</i> and <i>name-of-raster-map</i>.
+
+<p>Optionally, the user can define raster layers without labels. In that case
+only raster values are uploaded to a column with the name
+<i>name-of-raster-map</i>. 
+
+<p>Except the columns with raster values, the attribute table of the
+output vector point map can optionally include columns with coordinates.
+
+<p>The user can opt to include the attribute columns of the input vector layer
+in the output. In that case, the columns with the raster values and labels will
+appear after the columns from the input vector layer.
+    
```

Comment:
Would you mind adding an example preferably with North Carolina location or building the rasters from scratch and querying them?

Reply:
I'll add it